### PR TITLE
feat(dev): allows choice to be a dev or a user

### PR DESCRIPTION
**Pull request by voryzen**
Just needs the code changes, to make pre-commit less invasive for the user.
@denis-ev

### CODE REVIEW

**Comment by denis-ev** [comment]
@voryzen alright, will do that in a bit. Isn't too hard to achieve.

**Comment by denis-ev** [comment]
> Just needs the code changes, to make pre-commit less invasive for the user.

What do you exactly mean with that? 
I think you already changed some of that stuff and I fix the problem with pre-commit, that the task will fail if not found.

File: tasks/DeveloperTasks.sh [hunk 1]
```diff
@@ -6,3 +6,33 @@ Task::lint(){
   Task::run_docker pip install yamllint; find . -type f -name '*.yml' | sed 's|\./||g' | egrep -v '(\.kitchen/|\[warning\]|\.molecule/)' | xargs yamllint -c yamllint.conf -f parsable
 
 }
+
+Task::dev_setup(){
+  echo "Welcome to VivumLab Dev setup"
+  echo ""
+  read -p "Do you want to contribute to VivumLab (yes or no)?" contribute
+  case $contribute in
+  [Yy][Ee][Ss]|[Tt][Rr][Uu][Ee])
```
Comment (denis-ev) [inline tasks/DeveloperTasks.sh:15]: this does not seem to work, I had to type yes for it to work.
Could you check this? @voryzen

Comment (voryzen) [reply tasks/DeveloperTasks.sh:15]: > this does not seem to work, I had to type yes for it to work.
> Could you check this? @voryzen

you can type YES, Yes, YeS, yes, yES... whatever you want.
Same goes for true.

What were you expecting?

**Comment by voryzen** [comment]
> > Just needs the code changes, to make pre-commit less invasive for the user.
> 
> What do you mean exactly with that?
> I think you already changed some of that stuff and I fixed the problem with pre-commit, that the task will fail if not found.

Ok, if it doesn't REQUIRE pre-commit to run (doesn't fail if pre-commit is not installed), then that is fine. It should be ready to merge

**Comment by denis-ev** [comment]
I was expecting a y or n

**Comment by denis-ev** [comment]
Perfect, it won’t fail if pre-commit isn’t installed now.

**Review by denis-ev — DISMISSED** [review]

**Review by denis-ev — APPROVED** [review]

**Pull request merged**